src/Selenium2Library/keywords/_screenshot.py

```diff
@@ -19,14 +19,22 @@ def capture_page_screenshot(self, filename=None):
         the Robot Framework log file is written into. The `filename` is
         also considered relative to the same directory, if it is not
         given in absolute format. If an absolute or relative path is given
-        but the path does not exist it will be created. 
+        but the path does not exist it will be created.
 
-        `css` can be used to modify how the screenshot is taken. By default
-        the bakground color is changed to avoid possible problems with
-        background leaking when the page layout is somehow broken.
-        """
-        path, link = self._get_screenshot_paths(filename)
+        `index` with index argument it is possible to define custom filename
+        but each time get unique screen capture. Example if pabot is used
+        to run multiple test suites in paraller and correct screenshots should
+        be copied to log, then it is possible to use `filename` and
+        `index` argument to gether to create unique name for each screenshot.
 
```

Comment:
Hmm, the current code does not actually do this, but by quick thinking, doing this should not be too difficult. And at least in my mind, it would make life easier. 

So is this what we want, at least it is good for me (and makes my life even more simpler):
```
| Screen Capture | filename=${BROWSER}.png |
| Screen Capture | filename=${BROWSER}.png |
| File Should Exist  | ${OUTPUTDIR}${/}${BROWSER}.png |
| File Should Exist  | ${OUTPUTDIR}${/}${BROWSER}-1.png |
| Screen Capture |
| Screen Capture |
| File Should Exist  | ${OUTPUTDIR}${/}selenium-screenshot-2.png |
| File Should Exist  | ${OUTPUTDIR}${/}selenium-screenshot-3.png |
| Screen Capture | filename=overwrite-${BROWSER}.png | overwrite=${True} |
| Screen Capture | filename=overwrite-${BROWSER}.png | overwrite=${True} |
| File Should Exist  | ${OUTPUTDIR}${/}overwrite-${BROWSER}.png | 
| File Should Not Exist | ${OUTPUTDIR}${/}overwrite-${BROWSER}-4.png | 
```